benchmark_tests/test_multi_stage_dynamic_to_static/test_multistage_dynamic_to_static.py

```diff
@@ -0,0 +1,149 @@
+    solution_type = SolutionType.DYNAMIC
+
+    # Set up start and end time of calculation, time step and etc
+    time_integration = TimeIntegration(start_time=0.0,
+                                       end_time=0.25,
+                                       delta_time=0.005,
+                                       reduction_factor=1.0,
+                                       increase_factor=1.0,
+                                       max_delta_time_factor=1000)
+    convergence_criterion = DisplacementConvergenceCriteria(displacement_relative_tolerance=1.0E-12,
+                                                            displacement_absolute_tolerance=1.0E-6)
+    stress_initialisation_type = StressInitialisationType.NONE
+    solver_settings = SolverSettings(analysis_type=analysis_type,
+                                     solution_type=solution_type,
+                                     stress_initialisation_type=stress_initialisation_type,
+                                     time_integration=time_integration,
+                                     is_stiffness_matrix_constant=False,
+                                     are_mass_and_damping_constant=False,
+                                     convergence_criteria=convergence_criterion,
+                                     rayleigh_k=1.5e-3,
+                                     rayleigh_m=0.02)
+
+    # Set up problem data
+    problem = Problem(problem_name="test_multi_stage_dynamic_to_static", number_of_threads=2, settings=solver_settings)
+    model_stage_1.project_parameters = problem
+
+    # Define the results to be written to the output file
+    # Nodal results
+    nodal_results = [NodalOutput.DISPLACEMENT, NodalOutput.VELOCITY]
+
+    # Define the output process
+    model_stage_1.add_output_settings(output_parameters=VtkOutputParameters(file_format="ascii",
+                                                                            output_interval=1,
+                                                                            nodal_results=nodal_results,
+                                                                            gauss_point_results=[],
+                                                                            output_control_type="step"),
+                                      output_dir="output",
+                                      output_name="vtk_output")
+
+    # define the STEM instance
+    input_folder = "benchmark_tests/test_multi_stage_dynamic_to_static/inputs_kratos"
+    stem = Stem(model_stage_1, input_folder)
+
+    # create new stage
+    model_stage_2 = stem.create_new_stage(0.01, 0.1)
+
+    # Set up solver settings for the new stage
+    model_stage_2.project_parameters.settings.solution_type = SolutionType.QUASI_STATIC
+    model_stage_2.project_parameters.settings.rayleigh_k = 0.0
+    model_stage_2.project_parameters.settings.rayleigh_m = 0.0
+
+    # model_stage_2.process_model_parts[0].parameters.value = [0, -1050, 0]
```

Comment:
I think its better to keep this, as you can see if something changes, else dynamic and quasi static hold the same results